straxen/plugins/bayes_peaklet_classification.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import numba
+from scipy.special import logsumexp
+import strax
+import straxen
+
+
+class BayesPeakletClassification(strax.Plugin):
```

Comment:
is this based on peaklets or peaks? seems BayesPeakClassification is the right class name here?

Reply:
then, I think it's better to change it to `BayesPeakClassification` to avoid confusion.

Reply:
Ok, so then I will have to remove/add the file since is named `bayes_peaklet_classificaiton.py`

Reply:
@WenzDaniel @JoranAngevaare @skazama 
I guess we all agree to change the name?

Reply:
sounds good!

Reply:
done